### PR TITLE
Use python template to restructure the repo

### PR DESCRIPTION
**Pull request by geek-yang**
In this PR, we use the [python template](https://github.com/NLeSC/python-template) to restructure our repo.

### CODE REVIEW

File: docs/CHANGELOG.md [hunk 1]
```diff
@@ -0,0 +1,18 @@
...
+## [0.0.1] - 1900-12-31
+
+### Added
+
+### Removed
+
+### Changed
+
+[Unreleased]: https://github.com/olivierlacan/keep-a-changelog/compare/v1.0.0...HEAD
+[0.0.1]: https://github.com/olivierlacan/keep-a-changelog/releases/tag/v0.0.1
```
Comment (SarahAlidoost) [inline docs/CHANGELOG.md:18]:
```suggestion
[Unreleased]: https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/compare/v1.0.0...HEAD
[0.0.1]: https://github.com/EcoExtreML/STEMMUS_SCOPE_Processing/releases/tag/v0.0.1
```

**Review by SarahAlidoost — CHANGES_REQUESTED** [review]
@geek-yang nice work! I just left a comment to fix links. Also, I think we can remove the files `PyStemmusScope/my_module.py` and `tests/test_my_module.py` as they are just templates. 

Regarding readthedocs and sonar, later we enable them, for now the configurations look OK.

File: docs/CHANGELOG.md (hunk 1 above)
Comment (geek-yang) [reply docs/CHANGELOG.md:18]: Nice spot! I will change them.

**Comment by geek-yang** [comment]
> @geek-yang nice work! I just left a comment to fix links. Also, let's remove the files `PyStemmusScope/my_module.py` and `tests/test_my_module.py` as they are just templates.
> 
> Regarding readthedocs and sonar, later we enable them, for now the configurations look OK.

Thanks for your @SarahAlidoost comments! I update the links and remove those template files. But this will break the tests since now there is no testing stuff. I can fix it when working on #8 since we will create modules and test them.

**Review by SarahAlidoost — APPROVED** [review]

**Comment by SarahAlidoost** [comment]
> > @geek-yang nice work! I just left a comment to fix links. Also, let's remove the files `PyStemmusScope/my_module.py` and `tests/test_my_module.py` as they are just templates.
> > Regarding readthedocs and sonar, later we enable them, for now the configurations look OK.
> 
> Thanks for your @SarahAlidoost comments! I update the links and remove those template files. But this broke the tests since now there is no testing stuff. I can fix it when working on #8 since we will create modules and test them.

Thanks for addressing the comments. I agree, we will fix them when needed.

**Pull request merged**